wire-ios/Wire-iOS/Sources/UserInterface/ContactsUI/ContactsViewController+Constraints.swift

```diff
@@ -79,19 +77,21 @@ extension ContactsViewController {
             bottomContainerSeparatorView.heightAnchor.constraint(equalToConstant: 0.5)
         ]
 
-        let bottomEdgeConstraint = inviteOthersButton.bottomAnchor.constraint(equalTo: inviteOthersButton.superview!.bottomAnchor, constant: -(standardOffset / 2.0 + UIScreen.safeArea.bottom))
-        self.bottomEdgeConstraint = bottomEdgeConstraint
-
-        constraints += [
-            bottomEdgeConstraint,
-            inviteOthersButton.topAnchor.constraint(equalTo: inviteOthersButton.superview!.topAnchor, constant: standardOffset / CGFloat(2)),
-            inviteOthersButton.leadingAnchor.constraint(equalTo: inviteOthersButton.superview!.leadingAnchor, constant: standardOffset),
-            inviteOthersButton.trailingAnchor.constraint(equalTo: inviteOthersButton.superview!.trailingAnchor, constant: -standardOffset)
-        ]
+        if let superview = inviteOthersButton.superview {
```

Comment:
When is this condition supposed to be true and when false?
If it's always supposed to evaluate to true we might not want to fail silently.

Reply:
True when inviteOthersButton is already in the view hierarchy and has a parent view.
False when inviteOthersButton is not yet added to any parent view, and thus the superview is nil.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/33dda8e7939078d0aced079c80c6beaeeaa62189